### ISSUE TITLE
Accessibility: <article> tags lack heading.

### ISSUE THREAD

**Issue by alan-cole** [dpc-sdp/ripple#670]
WYSIWYG elements can contain`<article>` tags generated within Drupal. 
Cases such as `<article class="media media--type-image media--view-mode-embedded">` throw an [error](https://www.w3.org/wiki/HTML/Usage/Headings/Missing) in [HTML validators](https://validator.w3.org/nu) due to missing heading elements within the `<article>` tag.

Consider adding a new component to transpile Drupal embedded images within WYSIWYGs into custom components so the markup can be made compliant.

**Comment by dylankelly** [comment]
Thanks @alan-cole - I have some work to implement the image lazy loading that I did for VT in Ripple - Its tracked in SDPA-5196. I would expect that all images added via the wysiwyg should be striped of Drupal rendered markup and implemented with this component, resolving the issue raised.
